draft-ietf-oauth-status-list.md

```diff
@@ -130,9 +131,9 @@ Each status of a Referenced Token MUST be represented with a bit size of 1,2,4,
 
 1. The overall Status List is encoded as a byte array. Depending on the bitsize, each byte corresponds to 8/(#bit-size) statuses (8,4,2, or 1). The status of each Referenced Token is identified using the index that maps to one or more specific bits within the byte array. The index starts counting at 0 and ends with "size" - 1 (being the last valid entry). The bits within an array are counted from least significant bit "0" to the most significant bit ("7"). All bits of the byte array at a particular index are set to a status value.
 
-2. The complete byte array is compressed using gZIP {{RFC1952}}.
+2. The complete byte array is compressed using the "DEFLATE" {{RFC1951}} compression method and stored using the "ZLIB" {{RFC1950}} data format. Implementations are RECOMMENDED to use the highest compression level available.
```

Comment:
I'm not sure I understand "compressed using the "DEFLATE" {{RFC1951}} compression method and stored using the "ZLIB" {{RFC1950}} data format" and not sure I understand well enough to even ask appropriate questions. Does this say to compress twice, once with DEFLATE only and once with ZLIB? That sounds silly but could easily be read that way. Or like what does "stored" mean in this context? Stored where? This should be about data format for exchange between parties. Whereas storage typically implies something internal (which wouldn't be in scope for the interchange format). I would expect this draft to clearly specify one compression method. This reads like two compressions. The code below suggests that ZLIB is being used - shouldn't the text reflect that? I know zlib does use DEFLATE internally and that is confusing things but indicating them both with an "and" is further confusing.

Reply:
So Zlib is using Deflate as the compression method internally and zlib is more of a data format wrapping the result. "stored" may indeed be confusing here, is "encoded" better? Do you think its better not to reference Deflate as its referenced by Zlib RFC?

Some implementations may not offer the choice for the compression level, so I think that was the idea to not mandate the level of compression. Although we don't have extensive experience with many programming languages yet.

Reply:
The wording is tricky (for me anyway!) but maybe something like this kinda following the language example from HTTP conveys the intent here more clearly: 

```suggestion
2. The complete byte array is compressed using DEFLATE {{RFC1951}} inside the ZLIB {{RFC1950}} data format. Implementations are RECOMMENDED to use the highest compression level available.
```

maybe...

Reply:
This PR has been merged already so I created issue #94